code/modules/antagonists/abductor/machinery/camera.dm

```diff
@@ -69,6 +69,11 @@
 	var/mob/living/carbon/human/C = owner
 	var/mob/camera/aiEye/remote/remote_eye = C.remote_control
 	var/obj/machinery/abductor/pad/P = target
+	var/target_loc = get_turf(remote_eye)
+
+	if(istype(get_area(target_loc), /area/ai_monitored))
+		to_chat(owner, "<span class=warning>This area is coated in bluespace interference, it's not possible to warp here safely!</span>")
```

Comment:
How do you coat something in interference

Reply:
By polluting the epiglossic heliospectrum of the area, preventing brownian intercession

Reply:
Interference would be the result of a coating, not the coating itself

Reply:
i changed it maybe this is better lmao